Paralayout/UIViewAdditions_Distribution.swift

```diff
@@ -18,49 +18,68 @@ import UIKit
 
 /// A means of getting a `SubviewDistributionItem`: either a UIView, or a number as `.fixed` or `.flexible`.
 public protocol ViewDistributionSpecifying {
-    
+
     var distributionItem: ViewDistributionItem { get }
-    
+
 }
 
 // MARK: -
 
 /// A direction for subview distribution.
 public enum ViewDistributionAxis {
 
-    /// The horizontal direction, meaning views will be distributed left-to-right.
+    /// The horizontal direction, meaning views will be distributed left-to-right or right-to-left.
     case horizontal
```

Comment:
We could also attach `layoutDirection: UIUserInterfaceLayoutDirection` as an associated value of the `horizontal` case. Maybe, it would be easier to read?

Reply:
I think for the public API we'll always want it to match the receiver's effective user interface layout direction. We could potentially split into a public enum without an associated value and an internal enum with an associated value to clean up some of the logic, but I'm not sure it's worth the extra type complexity.